### ISSUE TITLE
Webpack-stream WITH Gulp WITH webpack-dev-server??

### ISSUE THREAD

**Issue by jsa41394** [shama/webpack-stream#121]
https://github.com/webpack/webpack-dev-middleware/issues/66
ERROR: compiler.plugin is not a function
Also using webpack-stream, but how else to integrate with Gulp?
See: https://webpack.github.io/docs/usage-with-gulp.html
Excerpt:
```
const webpack = require('webpack-stream');
const webpackConfig = require("./misc/webpack.config.js");
// ...
gulp.task("webpack-dev-server", function(callback) {
    // Start a webpack-dev-server
    var compiler = webpack(webpackConfig);
    new WebpackDevServer(compiler, {
        // server and middleware options
    }).listen(8080, "localhost", function(err) {
        if(err) throw new gutil.PluginError("webpack-dev-server", err);
        // Server listening
        gutil.log("[webpack-dev-server]", "http://localhost:8080/webpack-dev-server/index.html");

        // keep the server alive or continue?
        // callback();
    });
});
```

**Comment by jsa41394** [comment]
Nevermind, it works, except not webpack-stream

**Comment by mchambaud** [comment]
How did you solve this? I get the same error event with your code snippet.

**Comment by jsa41394** [comment]
I ended up using gulp-connect and gulp-util-loader. I made a working e.g. here:github.com/jsa41394/Front-End-Tools
Should take 5min to set-up (aside from npm install downloading). Pls post any issues and feel free to modify code.  
 
  On Mon, Jul 18, 2016 at 11:49 AM, Michael Chambaud<notifications@github.com> wrote:   
How did you solve this? I get the same error event with your code snippet.

**Comment by jsa41394** [comment]
this is what i was trying to do, maybe can modify with webpack:https://github.com/samwx/html-skeleton-gulpalso a sass version

mine was working libereload until react/js code had an error and gulp would just quit instead of handle error.


 
 
  On Mon, Jul 18, 2016 at 12:59 PM, jason.albalah@yahoo.com<jason.albalah@yahoo.com> wrote:   I ended up using gulp-connect and gulp-util-loader. I made a working e.g. here:github.com/jsa41394/Front-End-Tools
Should take 5min to set-up (aside from npm install downloading). Pls post any issues and feel free to modify code.  
 
  On Mon, Jul 18, 2016 at 11:49 AM, Michael Chambaud<notifications@github.com> wrote:   
How did you solve this? I get the same error event with your code snippet.
